pages/create/index.js

```diff
@@ -0,0 +1,311 @@
+import PetSelection from "@/Component/PetSelection/PetSelection";
+import { indicatorData, characteristicOptions, animalList } from "@/Lib/Data";
+import { useState } from "react";
+import styled from "styled-components";
+import Indicator from "@/Component/Indicator/Indicator";
+import Link from "next/link";
+
+const StyledCreatePage = styled.main`
+  background-image: var(--create-image);
+  height: 100%;
+  width: 100%;
+`;
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 30px;
+  padding: 10% 0;
+
+  @media screen and (min-width: 1024px) {
+    padding: 5% 0;
+  }
+`;
+
+const StyledConfirmButtonContainer = styled.section`
+  display: flex;
+
+  a {
+    display: grid;
+    place-content: center;
+    width: 25vw;
+    height: 10vh;
+    max-width: 200px;
+    font-size: 20px;
+    border-radius: 10px;
+    margin: 0 20px;
+    box-shadow: 2px 2px black;
+    cursor: pointer;
+    background-color: var(--signal-color);
+    background-image: var(--button-image);
+    color: #000000;
+    text-decoration: none;
+    font-family: sans-serif;
+    border: 1px solid #000000;
+
+    &:hover {
+      transform: scale(1.2);
+    }
+
+    &:active {
+      background-color: var(--secondary-color);
+    }
```

Comment:
This anchor tag has the same styling as the link on your starting page.

Make this a StyledComponent as use it in two places